<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qt_4dofClass</class>
 <widget class="QMainWindow" name="qt_4dofClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CHAI3D</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/chai3d.ico</normaloff>:/chai3d.ico</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="Settings" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupHapticLin">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Application Parameters</string>
         </property>
         <layout class="QGridLayout" name="gridLayoutHaptic">
          <item row="0" column="0">
           <widget class="QLabel" name="titleZoom">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="toolTip">
             <string>Scaling factor between haptic translation and robot translation</string>
            </property>
            <property name="text">
             <string>Zoom</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QSlider" name="sliderZoom">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="labelZoom">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>000.0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="xBox">
         <property name="title">
          <string>X Neutral Position</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="xMinusButton">
              <property name="text">
               <string>x-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="xSlider">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
              <property name="value">
               <number>50</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="xPlusButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>x+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="xPos">
              <property name="text">
               <string>000.0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
         <zorder></zorder>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="yBox">
         <property name="title">
          <string>Y Neutral Position</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="yMinusButton">
              <property name="text">
               <string>y-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="ySlider">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>50</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="yPlusButton">
              <property name="text">
               <string>y+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="yPos">
              <property name="text">
               <string>000.0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="zBox">
         <property name="title">
          <string>Z Neutral Position</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="zMinusButton">
              <property name="text">
               <string>z-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="zSlider">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>0</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="zPlusButton">
              <property name="text">
               <string>z+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="zPos">
              <property name="text">
               <string>000.0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="thetaBox">
         <property name="title">
          <string>Theta Neutral Position</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="thetaMinusButton">
              <property name="text">
               <string>theta -</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="thetaSlider">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="thetaPlusButton">
              <property name="text">
               <string>theta +</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="thetaPos">
              <property name="text">
               <string>000.0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
         <zorder></zorder>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionShow_Settings"/>
   <addaction name="actionFull_Screen"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionShow_Settings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionFull_Screen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Application.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionShow_Settings</sender>
   <signal>triggered(bool)</signal>
   <receiver>Settings</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
